pkg/workload/ycsb/ycsb.go

```diff
@@ -185,7 +185,7 @@ func (g *ycsb) Hooks() workload.Hooks {
 			var defaultReqDist string
 			incomingFreqSum := g.readFreq + g.insertFreq + g.updateFreq + g.deleteFreq + g.scanFreq + g.readModifyWriteFreq
 			if g.workload == "CUSTOM" {
-				if incomingFreqSum != 1.0 {
+				if incomingFreqSum > 1.00001 || incomingFreqSum < 0.99999 {
```

Comment:
Consider `if math.Abs(incomingFreqSum - 1.0) > 0.00001 {`. We use that approach elsewhere.